drivers/eeprom/i2c_xx24xx.c

```diff
@@ -472,7 +472,7 @@ static off_t ee24xx_seek(FAR struct file *filep, off_t offset, int whence)
     {
       filep->f_pos = newpos;
       ret = newpos;
-      finfo("SEEK newpos %d\n", newpos);
+      finfo("SEEK newpos %ld\n", (long)newpos);
```

Comment:
I think we have `PRIdOFF`

Reply:
Yes. I included inttypes.h and used PRIdOFF for off_t and %zu for size_t